README.md

```diff
@@ -157,3 +157,26 @@ Common usage:
   ```shell
   docker run --rm -v ~/solid-config:/config -p 3000:3000 -it css:latest -c /config/my-config.json
   ```
+
+## Using the identity provider
+
+1. First launch the Community Solid Server (see [Running locally](#running-locally)).
+2. To use the identity provider, you need a compatible client application.
+
+    You can use for example `@inrupt/solid-client-authn-js`:
+
+    ```bash
+    git clone https://github.com/inrupt/solid-client-authn-js
+    cd solid-client-authn-js
+    npm ci
+    cd packages/node/example/demoClientApp/
+    npm run start
+    ```
+
+    Go to `http://localhost:3001`.
+3. Use the base URL of your running CSS instance to as Identity provider, for
```

Comment:
I changed the `index.js` to edit the url, is that how you're supposed to do it?

Reply:
You can simply enter the URL in the UI.
But yes, adding it to index.js is the way to add it by default.

Reply:
I couldn't find how to do it through the UI, if I clicked the URL it tried to go to that website, and if I clicked login it tried to register with that provider 😅